src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="6.1.2" />
     <PackageReference Include="Swashbuckle.AspNetCore.Swagger" Version="6.1.2" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20574.7" />
+    <PackageReference Include="System.Text.Json" Version="6.0.4" />
```

Comment:
Could you update the version in THIRD_PARTY_LICENSES and the internal dependency tracking tool also?

Reply:
Done